hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineStateManagerV2Impl.java

```diff
@@ -78,7 +78,8 @@ private void initialize() throws IOException {
         iterator = pipelineStore.iterator();
     while (iterator.hasNext()) {
       Pipeline pipeline = iterator.next().getValue();
-      addPipeline(pipeline.getProtobufMessage());
```

Comment:
addPipeline will do the following:
```
      Pipeline pipeline = Pipeline.getFromProtobuf(pipelineProto);
      pipelineStore.put(pipeline.getId(), pipeline);
      pipelineStateMap.addPipeline(pipeline);
      nodeManager.addPipeline(pipeline);
```
`pipelineStore.put(pipeline.getId(), pipeline);` should be avoided.